app/Http/Requests/Api/UserRegisterRequest.php

```diff
@@ -24,7 +26,21 @@ public function authorize()
     public function rules()
     {
         return [
-            //
+            'full_name' => 'required|max:255',
+            'email' => 'required|email|max:255|unique:users',
+            'password' => 'required|min:6|confirmed',
         ];
     }
+
+    /**
+     * Get the proper failed validation response for the request.
+     *
+     * @param array $errors fail information of validation
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function response(array $errors)
```

Comment:
You should define this function on `extends` class. Don't need to repeat on each Request class.